Sources/Widgets/Widgets3D/ResliceCursorWidget/index.js

```diff
@@ -313,6 +314,43 @@ function vtkResliceCursorWidget(publicAPI, model) {
     publicAPI.modified();
   };
 
+  publicAPI.rotatePlane = (rotatedPlaneName, rotateAngle, rotatedAxisName) => {
```

Comment:
rotatedAxisName -> axisOfRotation ? (not sure about that though, it depends what the function documentation will say :-) )